### PR TITLE
updated  config.json to Azure Naming Conventions

### PR DESCRIPTION
**Pull request by precog90**
## Overview/Summary

Based on TBS feedback, Azure naming convention was not followed when creating the CaC resources. This PR provides an update to the default names used in the sample config.json file to reflect the Azure Naming convention listed here https://learn.microsoft.com/en-us/azure/cloud-adoption-framework/ready/azure-best-practices/resource-naming as per the cloud adoption framework

## Testing Evidence

Please provide any testing evidence to show that your Pull Request works/fixes as described and planned (include screenshots, if appropriate).

## As part of this Pull Request I have

Deployed the resources using the new naming convention:
image

- [ ] Checked for duplicate [Pull Requests](https://github.com/Azure/GuardrailsSolutionAccelerator/pulls)
- [ ] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/GuardrailsSolutionAccelerator/tree/main)
- [ ] Performed testing and provided evidence.

### CODE REVIEW

File: setup/config.json [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
-  "resourceGroup": "Guardrails",
+  "keyVaultName": "kvcac",
+  "resourceGroup": "rg-guardrails",
   "region": "CanadaCentral",
-  "storageAccountName": "guardrails",
-  "logAnalyticsworkspaceName": "guardrails",
-  "automationAccountName": "guardrails",
+  "storageAccountName": "stcac",
+  "logAnalyticsworkspaceName": "log-cac",
+  "automationAccountName": "aacac",
```
Comment (mbrat2005) [inline setup/config.json:7]: Would 'aa-cac' be better?

File: setup/config.json [hunk 2]
```diff
@@ -1,19 +1,19 @@
 {
-  "keyVaultName": "guardrails",
-  "resourceGroup": "Guardrails",
+  "keyVaultName": "kvcac",
```
Comment (mbrat2005) [inline setup/config.json:2]: Would 'kv-cac' be better?

**Review by mbrat2005 — COMMENTED** [review]
Good idea--I don't think we'd run into any issues adding delimiters, but will test the deployment

File: setup/config.json (hunk 2 above)
Comment (precog90) [reply setup/config.json:2]: yes, kv-cac will work..

**Review by mbrat2005 — APPROVED** [review]

**Pull request merged**